src/app.ts

```diff
@@ -122,4 +184,135 @@ app.get(
   }
 )
 
+app.get(
+  '/joinGame',
+  celebrate({
+    [Segments.QUERY]: Joi.object().keys({
+      playerToken: Joi.string().required().min(1).max(250).label('playerToken'),
+      nickname: Joi.string().required().max(20).label('nickname'),
+      gameId: Joi.number().required().min(0).max(999999).label('gameId'),
+    }),
+  }),
+  async (req, res) => {
+    let correctToken = true
+    if (process.env.JEST_WORKER_ID === undefined) {
+      // We don't want to verify tokens when testing
+      await verifyFCMToken(req.query.playerToken).catch((err) => {
+        console.error(err)
+        correctToken = false
+      })
+    }
+    if (!correctToken) {
+      return res.sendStatus(400)
+    }
+    const client = getClient()
+
+    client
+      .connect()
+      .then(async () => {
+        const checkIfGameExistsQuery = // todo add check if game is full
+          'SELECT game_master FROM Games WHERE game_id=$1'
+        const gameCheckValues = [req.query.gameId]
+        const createPlayerQuery =
+          'INSERT INTO Players(token, nickname, turn, game_id, card1, card2, funds, bet) VALUES($1, $2, $3, $4, $5, $6, $7, $8)'
+        const createPlayerValues = [
+          req.query.playerToken,
+          req.query.nickname,
+          0,
+          req.query.gameId,
+          null,
+          null,
+          null,
+          null,
+        ]
+        const getGameInfoQuery = 'SELECT * FROM Games WHERE game_id=$1'
+        const getGameInfoValues = [req.query.gameId]
+        const getPlayersInRoomQuery =
+          'SELECT nickname, token FROM Players WHERE game_id=$1'
+        const getPlayersInRoomValues = [req.query.gameId]
+        await client
+          .query(checkIfGameExistsQuery, gameCheckValues)
+          .then(async (result) => {
+            if (result.rowCount === 0) {
+              // game does not exist
```

Comment:
I think those comments don't add much, it's pretty clear what this if does

Reply:
Ok

Reply:
I removed them.